test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -97,13 +99,66 @@ func (data *testData) createPods(t *testing.T, ns string) error {
 	return err
 }
 
+func (data *testData) assertPodNetworkStatus(t *testing.T, clientset *kubernetes.Clientset, pods []*testPodInfo, ns string) error {
+	for _, pod := range pods {
+		if err := wait.PollUntilContextTimeout(context.Background(), time.Second, 20*time.Second, false, func(ctx context.Context) (done bool, err error) {
```

Comment:
I think it should use `assert.EventuallyWithT` with nested assertion, instead of mixing polling and assertion, which is inconsistent.

Reply:
done